tedana/selection/tedica.py

```diff
@@ -260,15 +209,14 @@ def selcomps(seldict, comptable, mmix, manacc, n_echos):
                     (comptable.loc[ncls, 'rho'] < rho_elbow)]
 
     if len(acc_prov) == 0:
-        LGR.warning('No BOLD-like components detected')
+        LGR.warning('No BOLD-like components detected. Ignoring all remaining '
```

Comment:
I think this is a very good tweak to the logger warning; inexperienced people may be confused when they see "ignored" components in the absence of it after a failure to detect BOLD-like components (I was, anyway). Nice job.